README.md

```diff
@@ -33,3 +33,13 @@ Prospective Collaborators:
 - https://github.com/codeisland/makermap
 
 To join us, please email our team at anansegroup@gmail.com
+
+Steps to get ready with the project
+- git clone <url>
```

Comment:
Here we can add the HTTP url to the master of the original repository of the ANANSE Group.